test_framework/scripts/longhorn-setup.sh

```diff
@@ -344,6 +344,18 @@ install_backupstores(){
   setup_azuitize_backup_store
 }
 
+install_backupstores_from_lh_repo(){
+  MINIO_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/minio-backupstore.yaml"
+  NFS_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/nfs-backupstore.yaml"
+  CIFS_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/cifs-backupstore.yaml"
+  AZURITE_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/azurite-backupstore.yaml"
+  kubectl create -f ${MINIO_BACKUPSTORE_URL} \
+                 -f ${NFS_BACKUPSTORE_URL} \
+                 -f ${CIFS_BACKUPSTORE_URL} \
+                 -f ${AZURITE_BACKUPSTORE_URL}
+  setup_azuitize_backup_store
+}
+
 setup_azuitize_backup_store(){
```

Comment:
The spelling of `azurite` is still incorrect after https://github.com/longhorn/longhorn-tests/pull/2155#issuecomment-2459186164. Could you help scan the codebase again and fix them?

Reply:
Update